.github/actions/issue-auto-label/src/index.ts

```diff
@@ -0,0 +1,34 @@
+import * as github from '@actions/github'
+import * as core from '@actions/core'
+
+const LABELS = {
+  LINEAR: 'linear',
+  KIND_BUG: 'kind: bug',
+}
+
+async function run() {
+  try {
+    const { payload, repo } = github.context
+    const {
+      issue,
+      pull_request,
+      label: { name: newLabel },
+    } = payload
+
+    if (pull_request || !issue?.body || !process.env.GITHUB_TOKEN) return
```

Comment:
you can return early if newLabel is not kind: bug

Reply:
I know I wanted to do this generic. I expect that we will have more label transformations.